entity-framework/core/extensions/index.md

```diff
@@ -312,3 +312,9 @@ Adds F# design-time support to EF Core. For EF Core: 5.
 Adds VB design-time support to EF Core. For EF Core: 5.
 
 [GitHub repository](https://github.com/efcore/EFCore.VisualBasic)
+
+### Krzysztofz01.EFCore.QueryFilterBuilder
+
+Extension for Entity Framework that allows you to create and manage multiple query filters.
```

Comment:
Please list the supported EF Core version(s), in the same format as all other extensions on this list.

Reply:
I added the supported version information in commit below.